docs/docs/domain.mdx

```diff
@@ -82,40 +82,73 @@ which can be used to store information the user provided (e.g their home city)
 as well as information gathered about the outside world (e.g. the result of a
 database query).
 
-Most of the time, you want slots to influence how the dialogue progresses.
-There are different slot types for different behaviors.
+Slots are defined in the `slots` section of your domain with their name and
+[type](./domain.mdx#slot-types). The slot type will determine the data type of the slot
+and how the slot
+[influences the assistant's behavior](.domain.mdx#slots-and-conversation-behavior).
+The following example defines a slot with name `slot_name` and the type `text`.
+
+```yaml
+slots:
+  # the name of the slot
+  slot_name:
+    # the type of the slot
+    type: text
+```
+
+### Slots and Conversation Behavior
 
+Most of the time, you want slots to influence how the dialogue progresses.
 For example, if your user has provided their home city, you might
-have a `text` slot called `home_city`. If the user asks for the
+have a slot called `home_city`. If the user asks for the
 weather, and you *don't* know their home city, you will have to ask
-them for it. A `text` slot only tells Rasa Core whether the slot
+them for it. A [text slot](domain.mdx#text-slot) only tells Rasa Core whether the slot
 has a value. The specific value of a `text` slot (e.g. Bangalore
 or New York or Hong Kong) doesn't make any difference.
 
-If you just want to store some data, but don't want it to affect the flow
-of the conversation, use an `unfeaturized` slot.
-If the value itself is important, use the [slot type](./domain.mdx#slot-types) that
-fits the type of behavior you want in your stories.
+```yaml
+slots:
+  # this slot will influence the conversation depending on
+  # whether the slot is set or not
+  home_city:
+    type: text
+    influence_conversation: true
+```
+
+By default all slot types will influence the conversation except slots of the type
+`any`. Depending on the [slot type](./domain.mdx#slot-types), a set slot will cause
+different behavior of your assistant so be careful which type you choose.
 
-Define your slot in your domain according to its slot type, following one
-of the examples below.
+
+If you want to store information but don't want a slot to influence the conversation,
+set the `influence_conversation` property of your slot to `false`.
+The following example defines a slot `age` which will store information about the
+user's age, but which will *not* influence the flow of the conversation. This means
+that the assistant will ignore the value of the slot during the conversation.
+
+```yaml
+slots:
+  age:
+    type: text
+    # this slot will not influence the predictions
+    # of the dialogue policies
+    influence_conversation: false
+
+```
 
 ### Slot Types
 
 Make this a nice table instead.
```

Comment:
I think a table would be too wide and hence annoying. Also, this way we can link to individual slot types which is nice.